favorite-sites.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My favorite websites</title>
+	<meta charset="utf-8">
+	<meta name="Description" content=This page show my favorite websites">
+	<meta name="keywords" content="best website, best java website, , best web development website">

```

Comment:
you've got an unnecessary comma. It should be removed

Reply:
Aah, It was done by mistake.